types/react-select/index.d.ts

```diff
@@ -263,6 +263,10 @@ export interface ReactSelectProps<TValue = OptionValues> extends React.Props<Rea
      * CSS className for the outer element
      */
     className?: string;
+    /**
+     * Prefix prepended to element default className if no className is defined
+     */
+    classNamePrefix?: string;
```

Comment:
Is it for react-select v2? Comment above:
// Type definitions for react-select 1.2